packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -8,8 +8,8 @@ export default class AnthosTenantsService {
     this.ovhDocUrl = ovhDocUrl;
   }
 
-  fetch(endPoint) {
-    return this.$http.get(endPoint).then(({ data }) => data);
+  fetch(endPoint, headers = undefined) {
```

Comment:
what is the point of providing "undefined" as a default value, won't it be undefined by default if not provided ?

Reply:
True, not required.